src/AppInstallerCommonCore/Public/AppInstallerLanguageUtilities.h

```diff
@@ -138,7 +138,7 @@ namespace AppInstaller
         const typename Variant::variant_t& GetVariant(Enum e) const
         {
             auto itr = m_data.find(e);
-            THROW_HR_IF_MSG(E_NOT_SET, itr == m_data.cend(), "GetVariant(%d)", e);
+            THROW_HR_IF_MSG(E_NOT_SET, itr == m_data.cend(), "GetVariant(%d)", static_cast<int>(e));

```

Comment:
@sreadingMSFT Apparently you were not the only one to see it even if it didn't give me any issues.

Reply:
Great. Glad the fix was easy enough, the build error was not particularly helpful.

Reply:
Ashwini and I both saw this yesterday after we merge with latest, we fixed it the same way. Looks like some new code called into this const path.